dot-analysis.cabal

```diff
@@ -19,7 +19,7 @@ maintainer:         guillaume.allais@ens-lyon.org
 -- A copyright notice.
 -- copyright:
 -- category:
-extra-source-files: CHANGELOG.md
+-- extra-source-files: CHANGELOG.md
```

Comment:
I wasn't able to install `dot-analysis` with this line uncommented -
```
saransh@Saranshs-MacBook-Air dot-analysis % cabal install .
Error: cabal: sdist of dot-analysis-0.1.0.0: filepath wildcard 'CHANGELOG.md'
does not match any files
```

Is there a betterway to get rid of this error? Commenting out the line makes it go away.